babybuddy/settings/base.py

```diff
@@ -12,6 +12,8 @@
 
 load_dotenv(find_dotenv())
 
+REVERSE_PROXY_AUTH = bool(int(os.getenv("REVERSE_PROXY_AUTH", False)))
```

Comment:
Is there a specific reason to use `bool` and `int` here? For example below for `DEBUG` we use `bool(strtobool(os.environ.get("DEBUG") or "False"))`. Can't remember why and don't know if one is better than the other. Just curious...

Reply:
Since I was largely referencing how Tandoor implemented it, there's not a preference. This is where my ignorance shows; if environment variables are always stored as strings, then sticking with strtobool would be better for consistency and wouldn't expect it to impact how a docker compose file is formatted.

I'll give it a try locally and see how it works.